bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,14 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% else %}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
```

Comment:
I definitely is confusing but the only fallback we have for right now. Are we good to merge with this as is? I will squash and merge once I get the 'ok' - Thanks everyone!